zp-relayer/router.ts

```diff
@@ -29,7 +29,6 @@ router.use((err: any, req: Request, res: Response, next: NextFunction) => {
 })
 
 router.get('/', endpoints.root)
-router.post('/sendTransaction', wrapErr(endpoints.sendTransaction))
 router.post('/sendTransactions', wrapErr(endpoints.sendTransactions))
 router.get('/transactions', wrapErr(endpoints.getTransactions))
```

Comment:
Should this be removed?

Reply:
Yes, this method is deprecated and not used by client library for a while. `/sendTransactions` basically does the same thing but can accept several transactions in one request.

Reply:
I meant this line:
```typescript
router.get('/transactions', wrapErr(endpoints.getTransactions))
```

Reply:
Got it. Same thing applies to this endpoint as well. Not used by client any more.

Reply:
Will remove it.